jsonnet/kube-prometheus/components/prometheus-adapter.libsonnet

```diff
@@ -236,13 +236,22 @@ function(params) {
         '--tls-cipher-suites=' + std.join(',', pa._config.tlsCipherSuites),
       ],
       resources: pa._config.resources,
+      startupProbe: {
+        httpGet: {
+          path: '/livez',
+          port: 'https',
+          scheme: 'HTTPS',
+        },
+        initialDelaySeconds: 30,
```

Comment:
is it really needed to delay the initial probing?

Reply:
Yeah, this shouldn't be needed and it would only delay the startup for smaller clusters